integration-tests.mdx

```diff
@@ -151,19 +151,30 @@ def test_transaction_aggregations(client):
 You can then run `pytest -s tests/test_transaction_aggregations.py`, where you will see a result like this:
 
 ```bash
-
-Chalk Feature Value Mismatch
-┏━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━┓
-┃ Kind   ┃ Name                        ┃ Value     ┃
-┡━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━┩
-│ Expect │ user.transaction_mean__3d__ │    20     │
-│ Actual │ user.transaction_mean__3d__ │    21     │
-└────────┴─────────────────────────────┴───────────┘
+$ pytest tests/test_transaction_aggs.py -s
+==================================================== test session starts =====================================================================
+platform darwin -- Python 3.10.14, pytest-8.3.3, pluggy-1.5.0
+rootdir: /Users/chalk/deployment/
+configfile: pyproject.toml
+plugins: anyio-4.4.0
+collected 1 item
+
+tests/test_transaction_aggs.py
+
+Chalk Feature Value Check Table
+┏━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━┓
+┃ Kind   ┃ Name                       ┃ Value ┃
+┡━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━┩
+│ Match  │ user.transaction_count[1d] │ 1     │
+│ Match  │ user.transaction_count[3d] │ 3     │
+│ Match  │ user.transaction_mean[1d]  │ 10.0  │
+│ Expect │ user.transaction_mean[3d]  │ 21.0  │
+│ Actual │ user.transaction_mean[3d]  │ 20.0  │
+└────────┴────────────────────────────┴───────┘
+{'user.transaction_count__259200__': 3, 'user.transaction_count__86400__': 1, 'user.transaction_mean__86400__': 10.0, 'user.transaction_mean__259200__': 20.0}
+F
```

Comment:
this F supposed to be here?

Reply:
yup, haha. It is a pytest and the test failed

Reply:
Though maybe it looks weird — I'll merge but can get rid of if we think it is better to in the future